Snakefile.genome

```diff
@@ -40,15 +41,20 @@ rule filter:
         sequences = lambda w: f"data/sequences_{subtype(w.build_name)}_{w.segment}.fasta",
         metadata = lambda w: f"data/metadata_{subtype(w.build_name)}.tsv",
         include = files.include,
+        exclude = files.dropped_strains
+    params:
+        min_date = "2024-01-01"
     output:
         sequences = "results/{build_name}/genome/sequences_{segment}.fasta"
     shell:
         """
         augur filter \
             --sequences {input.sequences} \
             --metadata {input.metadata} \
-            --exclude-all \
             --include {input.include} \
+            --exclude {input.exclude} \
+            --min-date {params.min_date} \
+            --query "(region == 'North America')" \
```

Comment:
This was me not understanding the best way to delimit Python strings, where it would be better to have `params.query` entry and then `--query {params.query}`.

Reply:
Try:

    --query {params.query:q}

see also https://docs.nextstrain.org/en/latest/reference/snakemake-style-guide.html#always-use-quoted-q-interpolations

Reply:
I was able to build this using Tom's suggestion + 
```
params:
     query = '(region == "North America")'` 
```
(note the switch of single & double quotes)

Reply:
Thanks for the pointer 🙏. Resolved in 2b3ef3d95fcd8a51eea3137f232bda8d010258fa.